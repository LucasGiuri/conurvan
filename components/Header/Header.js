import {Button} from './Header.styles';

const Header = ({children}) => (
	<div>
		<div>header</div>
		{children}
		<Button>111111122222</Button>
	</div>
);

export default Header;