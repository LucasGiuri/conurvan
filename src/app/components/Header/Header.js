import {Button} from './Header.styles';

const Header = ({children}) => (
	<div>
		header
		{children}
		<Button>111111122222</Button>
	</div>
);

export default Header;