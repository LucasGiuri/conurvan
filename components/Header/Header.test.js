import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

describe('Header', () => {
  it('Header basic render snapshoot', () => {
    const component = renderer.create(<Header children={<h2>1111</h2>}/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})
