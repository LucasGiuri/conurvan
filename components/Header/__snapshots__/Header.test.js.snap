// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Header basic render snapshoot 1`] = `
<div>
  <div>
    header
  </div>
  <h2>
    1111
  </h2>
  <button
    className="Headerstyles__Button-crkdu1-0 liizHN"
  >
    111111122222
  </button>
</div>
`;
